src/service/MilanApi.js

```diff
@@ -158,3 +159,31 @@ export const CreateEvent = async (eventdata) => {
     console.log(error);
   }
 };
+
+//* Google Auth screen
+
+export const GoogleAuth = async () => {
+  try {
+    const response = await Axios.get(`${API}/auth/google`);
+
+    return response.data.url;
+  } catch (error) {
+    toast.error(error, {
+      position: toast.POSITION.TOP_RIGHT,
+    });
```

Comment:
Can you please remove this code : 
```javascript
    toast.error(error, {
      position: toast.POSITION.TOP_RIGHT,
    });
```

and instead, use our personalised [toasts](https://github.com/tamalCodes/Milan/blob/main/src/utils/showToast.js) in the format `showSuccessToast("Hello")` or `showErrorToast ("Hello")`